scripts/redshift/launch_loader.py

```diff
@@ -3,8 +3,12 @@
 import getpass
 import os
 import subprocess
+import sys
 
-from ec2_instance_manager import EC2InstanceManager
+pkg_root = os.path.abspath(os.path.join(os.path.dirname(__file__), "..", ".."))  # noqa
+sys.path.insert(0, pkg_root)  # noqa
+
+from scripts.redshift.ec2_instance_manager import EC2InstanceManager
```

Comment:
Is this needed for mocking or something?

Reply:
It sets the runtime path to be `matrix-service/` regardless of where you run the script (previously, pre-unit tests, it would only be run from `matrix-service/scripts/`, for which the old import statement fails during unit tests)